k8_cortx_cloud/charts/cortx/templates/serviceaccount.yaml

```diff
@@ -0,0 +1,12 @@
+{{- if .Values.serviceAccount.create -}}
```

Comment:
This is just the default service account added by `helm create`. Near match to the current one in cortx-platform.